lib/presentation/screens/app_tutorial/app_tutorial_screen.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:animate_do/animate_do.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class SlideInfo {
+  final String title;
+  final String caption;
+  final String imageURL;
+
+  SlideInfo(this.title, this.caption, this.imageURL);
+}
+
+final slides = <SlideInfo>[
```

Comment:
Esto esta bueno, pero que te parece tener un archivo donde se manejen todos los String que se veran en la pantalla por ejemplo:

`Class WidggetTRanslations {
      static const String firstTitle = 'Search lunch';
      static const String firstInformation = 'Est ipsum in enim ullamco eiusmod enim qui enim adipisicing.';
      ...
}`

Reply:
el manejo de los strings lo hare mas adelante